packages-ts/starknet/src/account.ts

```diff
@@ -59,6 +60,7 @@ interface IFundingStrategy {
 // Fund the Account on Devnet
 class DevnetFundingStrategy implements IFundingStrategy {
   public async fund(accounts: FundAccounts[], opts: FunderOptions) {
+    const network = await startNetwork()
```

Comment:
The `DevnetFundingStrategy` should not be controlling the network, but just deal with the funding and assume the network is up already.